src/Sylius/Component/Core/Model/Order.php

```diff
@@ -110,6 +110,8 @@ class Order extends BaseOrder implements OrderInterface
      */
     protected $customerIp;
 
+    protected bool $byGuest = true;
```

Comment:
I'm not a fan of `byGuest`, but it is hard to propose something much better instead indeed. What would you say for one of these suggestions: 

This one seems to be the most readable:
```suggestion
    protected bool $guestOrder = true;
```

```suggestion
    protected bool $guest = true;
```

```suggestion
    protected bool $guested = true;
```

```suggestion
    protected bool $authrorized = true;
```

```suggestion
    protected bool $authrorizedOrder = true;
```

Reply:
To be honest, I'm also not fully convinced to `byGuest` but for me it seems to be the most readable version and none of your suggestions convinces me more 😢 maybe there should be sth like `createdByGuest`?
But for now, I will leave the current version

Reply:
`createdByGuest` is the winner